src/nix/eval.md

```diff
@@ -48,6 +48,13 @@ R""(
   # cat ./out/subdir/bla
   123
 
+* Replace evaluation errors:
```

Comment:
"Replace evaluation errors" is very vague (though the example does make it clear).

Reply:
It is clear when you have in mind the model of _evaluation as term rewriting_, although that's not how we introduce evaluation at all. In fact, we don't have a description of evaluation in the manual.

Reminded me of
- https://github.com/NixOS/nix/pull/13099

although that doesn't go all that much into term rewriting either.